logger.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, logger: logging.Logger):
     def __getattribute__(self: 'ExtendedLogger', name: str) -> typing.Any:
```

Comment:
Can we implement functions like `panic` and `log_once` ?

Reply:
to drop `__getattribute__()`? The last commit drops that.